subsys/net/lib/sockets/Kconfig

```diff
@@ -25,6 +25,13 @@ config NET_SOCKETS_POSIX_NAMES
 	attention, as in POSIX it closes any file descriptor, while with this
 	option enaled, it will still apply only to sockets.
 
+config NET_SOCKETS_POLL_NUM
```

Comment:
Recently introduce SYN backlog uses SIZE at all: CONFIG_NET_TCP_BACKLOG_SIZE. Anyway, I see that "MAX" is used in config options frequently, changing to it.